python/cudf/cudf/_lib/transform.pyx

```diff
@@ -52,6 +55,23 @@ def bools_to_mask(Column col):
     return buf
 
 
+def mask_to_bools(object mask_buffer, size_type begin_bit, size_type end_bit):
+    """
+    Given a mask buffer, returns a boolean column representng bit 0 -> False
+    and 1 -> True within range of [begin_bit, end_bit),
+    """
+    cdef bitmask_type* bit_mask
+    bit_mask= <bitmask_type*><uintptr_t>(mask_buffer.ptr)
```

Comment:
Should we validate that `mask_buffer` is an instance of `Buffer`?

Reply:
Yea, may be for better